app/clients/sms/aws_sns.py

```diff
@@ -66,12 +65,9 @@ def send_sms(self, to, content, reference, multi=True, sender=None):
             try:
                 start_time = monotonic()
                 response = client.publish(PhoneNumber=to, Message=content, MessageAttributes=attributes)
-            except botocore.exceptions.ClientError as e:
-                self.statsd_client.incr("clients.sns.error")
-                raise str(e)
             except Exception as e:
                 self.statsd_client.incr("clients.sns.error")
-                raise str(e)
+                raise Exception(e)
```

Comment:
Most likely it would be enough to re-raise the exception as-is as we do not add any new type, context or message to it? That should be less obfuscating as well instead of wrapping it up into a generic `Exception` type.
```suggestion
                raise e
```